src/ontology/mondo-edit.obo

```diff
@@ -434216,11 +434216,26 @@ id: MONDO:0100163
 name: SARS-CoV-2 Kawasaki-like syndrome
 def: "A inflammatory syndrome in children infected by the SARS-CoV-2 with similarities to Kawasaki disease. Clinical manifestations range from fever and inflammation to myocardial injury, shock, and development of coronary artery aneurysms." [PMID:32511692, PMID:32529358, PMID:32630212]
 comment: The clinical presentation of MIS-C includes fever, severe illness, and the involvement of two or more organ systems, in combination with laboratory evidence of inflammation and laboratory or epidemiologic evidence of SARS-CoV-2 infection. Some features of MIS-C resemble Kawasaki Disease, toxic shock syndrome, and secondary hemophagocytic lymphohistiocytosis/macrophage activation syndrome.
+synonym: "COVID-19 -related pediatric inflammatory multisystem syndrome" EXACT [NCIT:C172127]
 synonym: "COVID-19 Kawasaki-like syndrome" EXACT [https://orcid.org/0000-0002-0736-9199]
+synonym: "MIS-C" EXACT [PMID:32630212]
+synonym: "multisystem inflammatory syndrome in children" EXACT [DOID:0080711]
+synonym: "multisystem inflammatory syndrome in children associated with coronavirus disease 2019" EXACT [PMID:32768466]
+synonym: "multisystem inflammatory syndrome in children associated with COVID-19" EXACT [] {comment="PMID:32768466"}
+synonym: "multisystem inflammatory syndrome in children associated with COVID-19" EXACT [NCIT:C172127]
 synonym: "paediatric inflammatory multisystem syndrome: temporally associated with SARS-CoV-2" EXACT [PMID:32529358]
+synonym: "pediatric inflammatory multisystem syndrome" EXACT [NCIT:C172127]
 synonym: "pediatric inflammatory multisystem syndrome temporally associated with SARS-CoV-2" EXACT [PMID:32511692]
+synonym: "pediatric multi-system inflammatory syndrome potentially associated with COVID-19" EXACT [NCIT:C172127]
+synonym: "pediatric multisystem inflammatory syndrome" EXACT [NCIT:C172127]
+synonym: "PIMS" EXACT ABBREVIATION [NCIT:C172127]
 synonym: "PIMS-TS" EXACT ABBREVIATION [PMID:32511692, PMID:32529358]
+synonym: "PMIS" EXACT ABBREVIATION [NCIT:C172127]
+xref: DOID:0080711 {source="MONDO:equivalentTo"}
+xref: NCIT:C172127 {source="MONDO:equivalentTo"}
 is_a: MONDO:0100096 {source="PMID:32511692", source="PMID:32529358", source="PMID:32630212"} ! COVID-19
+relationship: has_modifier HP:0410280 {source="PMID:32511692", source="PMID:32529358", source="PMID:32630212"} ! Pediatric onset
+relationship: realized_in_response_to_stimulus NCBITaxon:2697049 {source="PMID:32511692", source="PMID:32529358", source="PMID:32630212"} ! Severe acute respiratory syndrome coronavirus 2
```

Comment:
not wrong, you already have this inherited from COVID.

we need to talk about our strategy for subtyping. If this is a subtype, what are its siblings?